addon/utils/missing-message.js

```diff
@@ -11,6 +11,14 @@ export default function missingMessage(key, locales) {
     return `No locale defined.  Unable to resolve translation: "${key}"`;
   }
 
+  if (options.default.length > 0) {
```

Comment:
Can you update https://github.com/ember-intl/ember-intl/blob/master/docs/missing-translations.md to cover this change